packages/openapi3/src/openapi.ts

```diff
@@ -1540,6 +1540,21 @@ function createOAPIEmitter(
     }
   }
 
+  function validateComponentFixedFieldKey(type: Type, name: string) {
+    const pattern = /^[a-zA-Z0-9\\.\-\\_]+$/;
```

Comment:
this allows `\`,
should be `const pattern = /^[a-zA-Z0-9\.\-_]+$/`

Reply:
hit the Error:   1544:33  error  Unnecessary escape character: \.  no-useless-escape